MapboxCoreNavigationTests/PassiveLocationDataSourceTests.swift

```diff
@@ -59,45 +60,52 @@ class PassiveLocationDataSourceTests: XCTestCase {
     }
     
     func testManualLocations() {
-//        let directions = DirectionsSpy()
+        let tilesVersion = "preloadedtiles" // any string
+
+        // Copy tiles to the cache directory, tiles should be placed in a folder with name
+        guard var tilesCacheURL = FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask).first else {
+            preconditionFailure("No Caches directory to create the tile directory inside")
+        }
+        if let bundleIdentifier = Bundle.main.bundleIdentifier ?? Bundle.mapboxCoreNavigation.bundleIdentifier {
+            tilesCacheURL.appendPathComponent(bundleIdentifier, isDirectory: true)
+        }
+        tilesCacheURL.appendPathComponent(".mapbox", isDirectory: true)
+        tilesCacheURL.appendPathComponent(tilesVersion, isDirectory: true)
+        let bundle = Bundle(for: Fixture.self)
+        let filePathURL: URL = URL(fileURLWithPath: bundle.bundlePath.appending("/tiles/liechtenstein"))
+        let fileManager = FileManager.default
+        var isDir : ObjCBool = false
+        if !fileManager.fileExists(atPath: tilesCacheURL.path, isDirectory:&isDir) {
+            do {
+                try fileManager.copyItem(atPath: filePathURL.path, toPath: tilesCacheURL.path)
+            }catch let error{
+                print(error.localizedDescription)
+            }
+        }
+
+        // Create PassiveLocationDataSource and configure it with the tiles version (version is used to find the tiles in the cache folder)
         let locationManager = PassiveLocationDataSource()
-        
-        let startingExpectation = expectation(description: "Location manager should start without an error")
-        locationManager.startUpdatingLocation { (error) in
-            XCTAssertNil(error)
-            startingExpectation.fulfill()
+        do {
+            try locationManager.configureNavigator(withTilesVersion: tilesVersion)
```

Comment:
If we can configure the PassiveLocationDataSource to use the Liechtenstein tiles, then we won’t need to call this private method. Avoiding an explicit call to this method would give us more confidence in the overall functionality in a more realistic setting.